build.sh

```diff
@@ -22,4 +22,4 @@ main () {
   docker tag "microdc/k8s-jenkins:${VERSION}" "microdc/k8s-jenkins:latest"
 }
 
-main "$@"
```

Comment:
Nothing is done with the positional variables, so I think it is less confusing to remove this